### DISCUSSION TITLE
Docsy & Hugo newbie question: Do I add docsy content using "hugo new"?

### DISCUSSION THREAD

**Discussion by cklann1** [google/docsy#693] [Q&A] [answered]
Hi -
I've cloned the docsy-example to a (docker-based) local instance to learn/create my own docsy-based site.

Being a newbie to both Hugo and docsy, I've skimmed both (great) documentations - particularly, the [docsy "adding content"](https://www.docsy.dev/docs/adding-content/content/) section, but I don't see an example of creating a new page is shown say, in the blog section.

It's my understanding that one uses the command line "hugo new <directory/page_name.md" command for new content but based on what I've read maybe the best method is to copy and existing file and update the frontmatter & content.

Thanks in advance!

**Comment by honzik20** [comment, marked as answer]
Hi @cklann1 and welcome.

You certainly can create a file with the command line, or you can copy an existing file and adapt the front matter and content. Perhaps the easiest way though is to simply create a new markdown file (.md) directly in your content folder. The only front matter it absolutely needs is `title`. I create a bunch of empty files in this way and play around with the table of contents structure before I get going.

**Reply by cklann1** [reply]
Thanks @honzik20 - great insights!
